src/ORM/Search/BasicSearchContext.php

```diff
@@ -33,28 +33,16 @@ class BasicSearchContext extends SearchContext
      *  If a filter is applied to a relationship in dot notation,
      *  the parameter name should have the dots replaced with double underscores,
      *  for example "Comments__Name" instead of the filter name "Comments.Name".
-     * @param array|bool|string $sort Field to sort on.
+     * @param array|false|string $sort Field to sort on.
```

Comment:
`true` isn't a valid value for `$sort` so I changed this to explicitly `false`. Not strong-typing here because the main focus of this PR is the `$limit` param.